pkg/connector/repo.go

```diff
@@ -69,15 +66,9 @@ func (r *repoBuilder) List(ctx context.Context, parentResourceID *v2.ResourceId,
 		PerPage: ITEMSPERPAGE,
 		Page:    pageToken,
 	})
+	err = getError(err)
```

Comment:
I think this one should also be `checkStatusUnauthorizedError()` to have the same behavior as the replaced code.

Reply:
Fixed